internal/bootstrap/agentbinary_test.go

```diff
@@ -51,24 +50,24 @@ func (s *agentBinarySuite) TestPopulateAgentBinary(c *tc.C) {
 
 	s.writeAgentBinary(c, toolsPath, current)
 
-	s.storage.EXPECT().Add(gomock.Any(), gomock.Any(), binarystorage.Metadata{
-		Version: current.String(),
-		Size:    size,
-		SHA256:  "sha256",
-	}).Return(nil)
+	//	s.storage.EXPECT().Add(gomock.Any(), gomock.Any(), binarystorage.Metadata{
```

Comment:
Can we just remove this whole test? I am 99% sure we don't need any of this and it all happens in the bootstrap worker now.